### PR TITLE
2252 change validation sidebar header text for the examples

### PR DESCRIPTION
**Pull request by jonfroehlich**
Resolves https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2252. Updates both the look and text of the "Example" headers on the right sidebar.

##### Before/After screenshots
**Before:**
image

image

**After:**
image

image

##### Things to check before submitting the PR 
- [x] If there is a visual design component, I've posted before/after screenshots on the relevant issue thread and have been given a thumbs up on my design.
- [x] I've asked for and added translations for any user facing text that was added or modified.
- [x] I've made sure my code and comments follow the [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide).
- [x] I've tested on multiple browsers.

##### Testing instructions
1. Just load any validation page

### CODE REVIEW

**Comment by misaugstad** [comment]
@jonfroehlich this is what the sidebar looks like when I check out your branch
Screenshot from 2020-09-25 19-49-20

**Comment by jonfroehlich** [comment]
I just checked and that's not what it looks like for me. Did you try clearing your cache?

image

image

**Comment by jonfroehlich** [comment]
Hey @misaugstad, I'd like to get this fixed and merged but I'm not seeing the problem you described. Can you test again and then provide me with details on how to replicate?

**Comment by misaugstad** [comment]
Not sure why they didn't show up for you. I didn't test again just now, but I peeked at the code, and it looks like it has to be from the code you (I assume accidentally) added to validation.scala.html

**Comment by jonfroehlich** [comment]
I just re-looked at my code, and I don't see anything wrong in there tbh. And it's not rendering incorrectly on my local dev machine.

File: app/views/validation.scala.html [hunk 1]
```diff
@@ -185,6 +185,7 @@ <h2 id="total-mission-reward" style="display:none"></h2>
                 </div>
                 <div class="status-box">
                     <h1 class="status-holder-header-1" id="label-type-example"></h1>
+                    <strong>Correct</strong> Examples
```
Comment (misaugstad) [inline app/views/validation.scala.html:188]: Did you mean to add these?

**Comment by jonfroehlich** [comment]
Thanks @misaugstad for the phone call to help explain my mistake. I moved stuff to messages. And fixed some minor aesthetic things. Ready for another review.

image

image

**Comment by misaugstad** [comment]
@jonfroehlich I don't know why you were not seeing the bug, but it was still there for me. The issue was that you added the updated text but you didn't remove the old text. You can see in my last two commits where I remove the old text.

**Comment by jonfroehlich** [comment]
My guess is I either forgot to commit my most recent changes or I was accidentally editing the file in /target, which has happened too often recently (and I don't notice unfortunately). But I had definitely solved the problem on my local dev machine.

(Note I've more recently realized that if a filename shows in orange in its editor tab then IntelliJ IDEA is indicating that it's coming from the /target area).

**Pull request merged**